src/storage-types/src/sources/encoding.rs

```diff
@@ -279,6 +279,21 @@ impl<C: ConnectionAccess> DataEncoding<C> {
         }
     }
 
+    pub fn type_(&self) -> &str {
+        match &self.inner {
+            DataEncodingInner::Avro(_) => "avro",
+            DataEncodingInner::Protobuf(_) => "protobuf",
+            DataEncodingInner::Csv(_) => "csv",
+            DataEncodingInner::Regex(_) => "regex",
+            DataEncodingInner::Bytes => "bytes",
+            DataEncodingInner::Json => "json",
+            DataEncodingInner::Text => "text",
+            // This is meant to be human-readable. In mz, postgres (and in the future, mysql),
+            // "row" describes the format we are reading the value as.
+            DataEncodingInner::RowCodec(_) => "row",
```

Comment:
Huh? Does this ever show up in the SQL catalog?

Reply:
yes, for pg (and in the future, mysql) sources

Reply:
oh and loadgen sources

Reply:
Would it be preferable to pass the source type into this function and match on that in this case?

Reply:
After talking to @petrosagg about this, we decided that `NULL` was the correct option here, because pg sources don't actually have a "format"